packages/xo-web/src/common/xo/copy-vm-modal/index.js

```diff
@@ -15,7 +15,7 @@ import { SelectSr } from '../../select-objects'
   {
     isZstdSupported: createSelector(
       createGetObject((_, { vm }) => vm.$container),
-      container => container.zstdSupported
+      container => container === undefined || container.zstdSupported
```

Comment:
So if the user can't see the container, you assume that zstd is supported, right?

Reply:
I not assume that it can be supported and i let the user choose if he want to use this compression or not due to the explicit option saying that it only supported on XCP-ng hosts

Reply:
Ok but the behaviour is the same as if it was supported.